packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -250,6 +261,12 @@ export default {
     },
 
     breadcrumbs() {
+      console.log(
```

Comment:
Do you want these merged?

Reply:
nope 😆 thanks for spotting

Reply:
Remove & force-merge? 😬

Reply:
Nope, I'm trying a different solution locally with a `watchActiveLocation` helper. Not sure if that'll work, yet. But the code change I made in this PR looks too verbose for potentially being applied in other locations.

Reply:
Please review again 😃